.test-infra/jenkins/jira_utils/jira_manager.py

```diff
@@ -175,9 +186,10 @@ def _append_descriptions(self, issue, dep_name, dep_latest_version):
         dep_name,
         dep_latest_version
     )
-    _, owners = self._find_owners(dep_name)
+    owners = self._find_owners(dep_name)
     for owner in owners:
       description += "[~{0}], ".format(owner)
+    description += "\n-------------- Do not modify the description above --------------\n"
```

Comment:
Regarding "Please review and upgrade the {1} to the latest version {2}", I think we wanted not to mention an specific version to upgrade to, right ?

How about just giving most up to date information in the JIRA description so that users can take an informed decision ? For example,

"Please consider upgrading dependency <>". Current version <> is <> months/years old and latest version is <>."

Reply:
Modified the description. For example:

------------------------- 2018-09-24 22:17:40.903859 -------------------------
Please consider upgrading the dependency group0:dep0.
The current version is 1.0.0. The latest version is 1.6.0
cc: yifanzou
Please refer to [Beam Dependency Guide](https://beam.apache.org/contribute/dependencies/) for more information. 
Do Not Modify The Description Above.